src/styles/_body.scss

```diff
@@ -1,3 +1,66 @@
+/* || HERO */
+
+//background for hero callout
+.usa-hero__callout {
+  @include u-bg('base-dark');
+}
+
+/* || TAGLINE */
+
+.tagline__heading {
+  .font-heading-xl,
+  .tagline__heading-text {
+    margin: 0;
+  }
+}
+
+/* || GRAPHICS SECTION */
+
+.graphic__image-container {
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+//circular images in graphics section
+.graphic__image--circle {
+  border-radius: 50%;
+}
+
+//graphics section image and text
+.usa-media-block {
+  padding: units(3) 0;
+}
+
+//for mobile up to tablet view the images are above the text
```

Comment:
At first, I didn't understand what you were saying here because of "view," maybe add a comma, colon, or rearrange the sentence.